lib/IRGen/GenType.cpp

```diff
@@ -1660,13 +1660,15 @@ IRGenModule::createNominalType(ProtocolCompositionType *type) {
 
   SmallVector<ProtocolDecl *, 4> protocols;
   type->getAnyExistentialTypeProtocols(protocols);
-
-  typeName.append("protocol<");
-  for (unsigned i = 0, e = protocols.size(); i != e; ++i) {
-    if (i) typeName.push_back(',');
-    LinkEntity::forNonFunction(protocols[i]).mangle(typeName);
+  
+  if (protocols.empty()) {
+    typeName.append("swift.Any");
```

Comment:
I think you want to use "Any" rather than "swift.Any" here? It's a language-specified type, so it doesn't really live in any namespace.